### ISSUE TITLE
Formatting curly brace

### ISSUE THREAD

**Issue by pizza-777** [tact-lang/tact-vscode#11]
Hi!
Now after formatting I have:
```
       send(SendParameters{to: to,
                value: ton("0.01"),
                body: "Hello from Tact!".asComment() 
            }
        );
```
But It would be better:
```
      send(SendParameters { //space before curly brace and param at new line
                to: to,
                value: ton("0.01"),
                body: "Hello from Tact!".asComment()
            }
       );
```

**Comment by pizza-777** [comment]
P.S. Join this repo to https://github.com/alefmanvladimir/HACK-TON-BERFEST-2023#-how-to-join-with-your-project

**Comment by logvik** [comment]
Thank you for the contribution!

**Comment by logvik** [comment]
fixed on v1.0.8
